src/java.base/share/lib/security/default.policy

```diff
@@ -163,6 +163,10 @@ grant codeBase "jrt:/jdk.internal.le" {
     permission java.security.AllPermission;
 };
 
+grant codeBase "jrt:/jdk.internal.vm.ci" {
```

Comment:
This is required as JVMCI is no longer loaded by the boot loader but should retain all permissions.